lib/sql/get.js

```diff
@@ -143,6 +146,28 @@ class Get {
     let url = sql.sqlURL(this.url, this.db, this.appId, 'crud', `${this.table}/read`);
     return fetchAsync(url, this.options);
   }
+
+  apply(){
+    this.options.body = JSON.stringify(this.params);
+    let url = sql.sqlURL(this.url, this.db, this.appId, 'crud', `${this.table}/read`);
+    return fetchAsync(url, this.options);
+  }
+
+  op(op){
+    if(op === 'one'){
+      this.params.op = 'one';
+
+      // Set a default limit if offset is specified and limit is not specified.
+      if (this.params.options.skip !== undefined && this.params.options.limit === undefined) {
```

Comment:
Checking the limits and offset should always be done in the apply method because there i no guarantee that `op` method will be called